components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/DataFlowProcessingEngine.cs

```diff
@@ -159,6 +161,39 @@ public class DataFlowProcessingEngine : IProcessingEngine, IDisposable {
             return Task.CompletedTask;
         }
 
+        /// <inheritdoc/>
+        public Task<JobDiagnosticInfoModel> GetDiagnosticInfo() {
```

Comment:
Does the method need to return `Task<>` ? Seems like it is a synchronous method, so maybe we can just return `JobDiagnosticInfoModel` ?